Framework/Sources/UI/Cells/BooleanTweakTableViewCell.swift

```diff
@@ -32,6 +35,26 @@ internal class BooleanTweakTableViewCell: UITableViewCell, TweakViewControllerCe
             detailTextLabel?.text = newValue
         }
     }
+
+    var feature: String? {
```

Comment:
Unless I'm missing something, I think you can just define
```
var feature: String?
var variable: String?
```

in both cells and remove the private variables with the underscores